jicoco-metrics/src/main/kotlin/org/jitsi/metrics/MetricsContainer.kt

```diff
@@ -110,17 +113,23 @@ open class MetricsContainer @JvmOverloads constructor(
         /** the optional initial value of the metric */
         initialValue: Long = 0
     ): CounterMetric {
-        if (metrics.containsKey(name)) {
+        val newName = when (name.endsWith("_total")) {
```

Comment:
Why use "when" instead of "if"?

Reply:
I think I was playing around and forgot to revert it to "if". Should I go ahead and change it to this?
```kotlin
val newName = if (name.endsWith("_total")) name else "${name}_total".also {
    logger.warn("Counter '$name' was renamed to '$it' to ensure consistent metric naming.")
}
```
Also, it currently logs whenever a metric is renamed (which currently applies to all of our CounterMetric). I think it's better to log this behavior even though it's the most common, as opposed to logging whenever a metric name is unchanged.